agent/cmd/checkPermission.go

```diff
@@ -189,7 +229,7 @@ func checkInterfaces(remoteHost string) bool {
 }
 
 // getCertInfo reveives a serialNumber and check on GSH API for certificate
-func getCertInfo(serialNumber string, keyID string, api string) (CertInfo, error) {
+func getCertInfo(serialNumber string, keyID string, keyFingerprint string, cert string, certType string, api string) (CertInfo, error) {
```

Comment:
Just to mention, as all `getCertInfo` parameters are of the type `string`, I would declare this way:

`func getCertInfo(serialNumber, keyID, keyFingerprint, cert, certType, api string)`

Reply:
Ok, corrected!